src/ducks/Signals/signalModal/SignalMasterModalForm.js

```diff
@@ -132,10 +132,17 @@ const SignalMasterModalForm = ({
               {...dialogProps}
             >
               <Dialog.ScrollContent className={styles.TriggerPanel}>
-                {isLoading && (
+                {isError && (
+                  <div className={styles.notSignalInfo}>
+                    Trigger with id {triggerId} does not exist or it is a
+                    private trigger owned by another user :(
+                  </div>
```

Comment:
suggest to reduce text and add image (like for watchlists):
![image](https://user-images.githubusercontent.com/24521041/62780771-1ed05380-babf-11e9-8e85-cf422836af39.png)